src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_drawing_2018_animation_model3d.g.cs

```diff
@@ -454,7 +454,7 @@ public OfficeArtExtensionList(string outerXml) : base(outerXml)
         internal override void ConfigureMetadata(ElementMetadata.Builder builder)
         {
             base.ConfigureMetadata(builder);
-            builder.SetSchema(112, "extLst");
+            builder.SetSchema(114, "extLst");
```

Comment:
Have you done the back and forth for this? We should really switch it out to use the proper schemas

Reply:
these were offset by retroactive introduction of the moniker types for drawing. I don't think they'll impact the public API surface or at least with any breaking change.